src/_test_/user.test.js

```diff
@@ -0,0 +1,19 @@
+import supertest from 'supertest'
+import { MongoMemoryServer } from 'mongodb-memory-server-core'
+import app from '../server.js'
+
+const request = supertest(app)
+
+
+describe('POST /api/register', () => {
+    it('responds with json', async () => {
+        const response = await request.post('/api/register')
```

Comment:
Codacy has a fix for the issue: [Replace `········const·response·=·await·request` with `····const·response·=·await·request⏎······`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
    const response = await request
```